mountpoint-s3/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-## Unreleased
+## v1.0.2 (September 22, 2023)
+
+### Breaking changes
+* No breaking changes.
+
+### Other changes
+* New Mountpoint releases are built on CentOS 7 instead of Amazon Linux 2. This lowers the minimum requirement to run Mountpoint to glibc 2.17 or newer. ([#517](https://github.com/awslabs/mountpoint-s3/pull/517))
+* Fixed a bug where writing to a file for longer than five minutes will result in a panic. ([#513](https://github.com/awslabs/mountpoint-s3/pull/513))
+* Updated the prefetcher to cancel discarded tasks and free up some unused resources on random read workloads. ([#505](https://github.com/awslabs/mountpoint-s3/pull/505))
+* Updated the CRT submodules to the latest releases. This includes fixing an issue with internal resource cleanup which could lead to Mountpoint hanging after a high number of file uploads. ([#529](https://github.com/awslabs/mountpoint-s3/pull/529))
```

Comment:
As @jamesbornholt suggested, should we just leave out the CRT update and focus on the fix? E.g.
```suggestion
* Fixed an issue with internal resource cleanup which could lead to Mountpoint hanging after a high number of file uploads. ([#529](https://github.com/awslabs/mountpoint-s3/pull/529))
```